### DISCUSSION TITLE
How to - actually - delete old archives

### DISCUSSION THREAD

**Discussion by kalipso-cyber** [borgbase/vorta#2081] [FAQ]
Hey,

I'm not sure if this is a bug or if I'm misunderstanding the concept. Searching the Interwebs and the issues here on Github did not help, so I hope somebody help me understand how this works.

Basically, I ran out of space on my backup disk, so I decided to manually delete some old archives (via the Vorta GUI). This went well in that Vorta seems to have deleted the archives (they don't appear in the Archive list anymore), but looking at the backup drive in the file explorer, it still only shows me 9,4GB of free space (despite having deleted literally hundreds of GBs in archives).

So now I can't backup any data until I manage to free up space on the drive, which I can't do manually due to the Vorta file structure, and which I cannot do via the Vorta GUI as it doesn't actually do anything in that regard. Pruning archives in Vorta does not seem to have any effect (in this regard) either, so now I'm a little lost. Do I need to format the drive? That would mean losing access to all my backups, which would be less than ideal.

I'd imagine that if I delete say 150GB worth of archives via Vorta, then I'd get around 150GB of newly freed up space on my backup disk. Is this a false assumption?

**Comment by m3nu** [comment]
1. Delete Archive
2. Compact repo (if it's a recent Borg version) to free the space.

**Comment by goebbe** [comment]
@kalipso-cyber  Vorta uses borgbackup in the background. If you delete an archive from a repository, the disc space is not released automatically. 
In Vorta, go to the Archives tab, at the bottom you will find a "Compact" key. You have to press the compact key to free the disc space. 

Keep in mind that with Vorta the archives are incremental. That is, if you do a new backup, only the changed files/ segments will be updated. This implies that if you delete an archive, only those files can be deleted that are unique to this archive. The files that did not change between backups (archives) are not deleted, since they are still part of the other archives.

**Reply by kalipso-cyber** [reply]
Oh I see, thank you for explaining!
